test/d2q/test/example/persons.clj

```diff
@@ -52,7 +52,7 @@
 ;; or ElasticSearch.
 
 ;; We will build a d2q Query Server which serves the following Fields,
-;; which names should be most self-explanatory:
+;; whose names should be mostly self-explanatory:
```

Comment:
@green-coder Fields are not persons, so I think 'which' is the  correct English word here :)

```suggestion
;; which names should be mostly self-explanatory:
```

Reply:
Oops, you may be right :-)
Let me re-check and I will fix-squash the commits.